x/rollup/keeper/msg_server.go

```diff
@@ -230,6 +195,61 @@ func (k *Keeper) SetL1BlockHistory(ctx context.Context, info *derive.L1BlockInfo
 	return nil
 }
 
+// processL1SystemDepositTx processes the L1 system deposit tx and returns the L1 block info.
```

Comment:
Maybe: `L1 Attributes deposit tx` for a tighter reference to [spec definitions](https://specs.optimism.io/protocol/deposits.html#l1-attributes-deposited-transaction).